### PR TITLE
[FIX] novuln check for ent query

### PR DESCRIPTION
**Pull request by pxp928**
# Description of the PR

fixes https://github.com/guacsec/guac/issues/1889

# PR Checklist

- [x] All commits have [a Developer Certificate of Origin (DCO)](https://wiki.linuxfoundation.org/dco) -- they are generated using `-s` flag to `git commit`.
- [x] All new changes are covered by tests
- [] If GraphQL schema is changed, `make generate` has been run
- [] If OpenAPI spec is changed, `make generate` has been run
- [ ] If `collectsub` protobuf has been changed, `make proto` has been run
- [x] All CI checks are passing (tests and formatting)
- [x] All dependent PRs have already been merged

### CODE REVIEW

**Review by mihaimaruseac — APPROVED** [review]

File: cmd/guacingest/cmd/ingest.go [hunk 1]
```diff
@@ -100,7 +100,7 @@ func ingest(cmd *cobra.Command, args []string) {
 			if errors.As(err, &urlErr) {
 				return fmt.Errorf("unable to ingest document due to connection error with graphQL %q : %w", d.SourceInformation.Source, urlErr)
 			}
-			logger.Errorf("unable to ingest document %q : %v", d.SourceInformation.Source, err)
+			d.ChildLogger.Errorf("unable to ingest document %q : %v", d.SourceInformation.Source, err)
```
Comment (pxp928) [inline cmd/guacingest/cmd/ingest.go:103]: note: this was a driveby fix as this needs to be the childlogger to retain proper output log when it errors.

**Review by mrizzi — APPROVED** [review]
@pxp928 thanks!
